cluster/examples/kubernetes/ceph/osd-purge.yaml

```
@@ -28,7 +28,8 @@ spec:
           image: rook/ceph:master
           # TODO: Insert the OSD ID in the last parameter that is to be removed
           # The OSD IDs are a comma-separated list. For example: "0" or "0,2".
-          args: ["ceph", "osd", "remove", "--osd-ids", "<OSD-IDs>"]
+          # If you want to preserve the OSD PVCs, set `--preserve-pvc=true`.
+          args: ["ceph", "osd", "remove", "--preserve-pvc=false", "--osd-ids", "<OSD-IDs>"]
```

I believe it works both ways, but for consistency with `--osd-ids` should we separate the arg?
```suggestion
          args: ["ceph", "osd", "remove", "--preserve-pvc", "false", "--osd-ids", "<OSD-IDs>"]
```